applications/app/templates/webAppManifest.scala.js

```diff
@@ -1,34 +1,71 @@
 @import conf.Static
 
-@()
-{
-    "name":"The Guardian",
-    "short_name":"The Guardian",
-    "icons":[
+@()(implicit request: RequestHeader)
+
+@if(experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
     {
-        "type": "image/png",
-        "sizes": "192x192",
-        "src": "@{JavaScript(Static("images/favicons/roundel-192x192.png"))}"
+        "name":"The Guardian",
+        "short_name":"The Guardian",
+        "icons":[
+        {
+            "type": "image/png",
+            "sizes": "192x192",
+            "src": "@{JavaScript(Static("images/favicons-garnett/roundel-192x192.png"))}"
     },
+        {
+            "type": "image/png",
+            "sizes": "152x152",
+            "src": "@{JavaScript(Static("images/favicons-garnett/roundel-152x152.png"))}"
+        },
+        {
+            "type": "image/png",
+            "sizes": "114x114",
+            "src": "@{JavaScript(Static("images/favicons-garnett/roundel-114x114.png"))}"
+        }
+    ],
+        "theme_color": "#005689",
```

Comment:
theme color is old guardian blue, do we need to change that?

Reply:
(i think it's the colour of tabs in android?)

Reply:
ah more than that https://developers.google.com/web/updates/2015/08/using-manifest-to-set-sitewide-theme-color

Reply:
Definitely lets change this! It doesn't looks right with the new styles anymore.

Reply:
white with black text?

Reply:
Or the grey/blue of the header. We can ask @blongden73 and @zeftilldeath and change tomorrow